include/NovelRT.h

```diff
@@ -28,6 +27,11 @@
 #include <type_traits>
 #include <vector>
 
+#if defined(_WIN32) || defined(_WIN64)
+#define WIN32_LEAN_AND_MEAN
```

Comment:
This is a very uhh, creative define? could you tell me why this name and why were including Windows.h for windows builds?

Reply:
`WIN32_LEAN_AND_MEAN` is just a special define that tells `<Windows.h>` to not include the world

We need to define it ourselves due to how glad defines some of its own macros if `Windows` was not included yet.